$OpenBSD: patch-include_ptlib_unix_ptlib_pmachdep_h,v 1.1 2011/10/26 16:59:09 ajacoutot Exp $

OpenBSD has support for monotonic clock (CLOCK_MONOTONIC), however we
currently don't have most of the _POSIX_* defines in unistd.h (e.g.
_POSIX_MONOTONIC_CLOCK 200112L) and the associated sysconf(3)
_SC_MONOTONIC_CLOCK variable.

--- include/ptlib/unix/ptlib/pmachdep.h.orig	Sat Oct 22 18:26:52 2011
+++ include/ptlib/unix/ptlib/pmachdep.h	Sat Oct 22 18:27:17 2011
@@ -115,6 +115,7 @@ typedef int socklen_t;
 #include <netinet/tcp.h>
 
 #define HAS_IFREQ
+#define _POSIX_MONOTONIC_CLOCK
 
 ///////////////////////////////////////////////////////////////////////////////
 #elif defined(P_NETBSD)
